assets/dev/js/frontend/utils/utils.js

```diff
@@ -13,7 +13,7 @@ export const escapeHTML = ( str ) => {
 
 // Check if Scroll-Snap is active.
 export const isScrollSnapActive = () => {
-	const scrollSnapStatus = elementorFrontend.isEditMode() ? elementor.settings.page.model.attributes.scroll_snap : elementorFrontend.config.settings.page.scroll_snap;
+	const scrollSnapStatus = elementorFrontend.isEditMode() ? elementor.settings.page.model.attributes?.['scroll_snap'] : elementorFrontend.config.settings.page?.['scroll_snap'];
```

Comment:
When I try to use  `elementor.settings.page.model.attributes?.scroll_snap` I get eslint error:
Identifier 'scroll_snap' is not in camel case.eslint
So I had to use `?.['scroll_snap']`